codeface/VCS.py

```diff
@@ -429,20 +429,38 @@ def get_feature_lines_from_file(file_layout_src, filename):
     # run cppstats analysis on the file to get the feature locations
     cmd = "/usr/bin/env cppstats --kind featurelocations --file {0} {1}"\
         .format(srcFile.name, featurefile.name).split()
-    execute_command(cmd, direct_io=True)
-
-    results_file = open(featurefile.name, 'r')
-    sep = parse_sep_line(next(results_file))
-    headlines = parse_line(sep, next(results_file))
-    feature_lines = \
-        get_feature_lines(
-            [parse_feature_line(sep, line) for line in results_file],
-            filename)
-
-    # clean up temporary files
-    srcFile.close()
-    featurefile.close()
+    try:
+        # direct_io is a bit noisy, but there are "silent" errors in
+        # cppstats which are only shown in its output
+        # and not in the return code. Enable the following line if
+        # you think you run into such an error (IE unit tests fail).
+        #execute_command(cmd, direct_io=True)
```

Comment:
It can, but I left it there on purpose. If you ever need to debug some "invisible" cppstats bug you will be thankful :) ...